### ISSUE TITLE
Few changes in the groups

### ISSUE THREAD

**Issue by Newfpv** [n3d1117/chatgpt-telegram-bot#138]
Add extra. setting for groups so that the bot translates voice messages into text, but does not respond to them

**Comment by n3d1117** [comment]
Hi @Newfpv, isn't `VOICE_REPLY_WITH_TRANSCRIPT_ONLY` what you're looking for? If you set it to true it will only answer with the transcript

**Comment by Newfpv** [comment]
Yes, but he will only respond in private messages with transcripts. I would like to communicate with him using voice in private messages

**Comment by stanislavlysenko0912** [comment]
@n3d1117 as I understand `IGNORE_GROUP_TRANSCRIPTIONS` already exists, can you close the issue?
